streams/src/main/java/org/apache/kafka/streams/processor/assignment/TaskInfo.java

```diff
@@ -50,6 +50,12 @@ public interface TaskInfo {
      */
     Set<String> stateStoreNames();
 
+    /**
+     *
+     * @return The topic partitions in use by this task.
+     */
+    Set<TaskTopicPartition> topicPartitions();
```

Comment:
This is temporary to get things to compile until we merge part 10 https://github.com/apache/kafka/pull/16034/files